src-ghc/Pact/Server/PactService.hs

```diff
@@ -99,7 +99,8 @@ applyCmd logger conf dbv gasModel bh _ pbh spv exConfig exMode _ (ProcSucc cmd)
   blocktime <-  (((*) 1000000) <$> systemSeconds <$> getSystemTime)
 
   let payload = _cmdPayload cmd
-      gasEnv = GasEnv (_pmGasLimit pubMeta) (_pmGasPrice pubMeta) gasModel
+      gasLimit = gasLimitToMilliGasLimit (_pmGasLimit pubMeta)
```

Comment:
Renaming the left hand side to `milliGasLimit` (and updating the rest of this function to match) would be slightly more internally consistent.

Reply:
I'm fine leaving the units out of this, we use the variable literally in the next line and the function call is clear. 

In general I'm okay with using something like `millis` suffix if we're mixing the units (e.g `Gas.hs`), but writing something like

```
gasLimitMillis = gasLimitToMilliGasLimit (_pmGasLimit pubMeta)
```
Is awfully close to hungarian notation and we specifically wrap our stuff in a newtype to have the compiler be the one yelling at us about this, it's also why `MilliGas` does not have a `Num` instance either, so we can't simply use literals to compare.